### ISSUE TITLE
Suggestion: default config path

### ISSUE THREAD

**Issue by ralphcallaway** [ChuckJonas/ts-force-gen#2]
If there is a standard form and path for the config file, perhaps create it as a default?

I.e. `ts-force-gen` would be equivalent to `ts-force-gen -j ts-force-config.json`

**Comment by ChuckJonas** [comment]
good idea... complicated by the fact that the command can be run without a json config file

**Comment by ralphcallaway** [comment]
oh interesting, what's the behavior when you don't pass in a config file?

**Comment by ChuckJonas** [comment]
IDK... there was a time where it made sense (using it in the [Test CI processes](https://github.com/ChuckJonas/ts-force/blob/59ffa8472968f49ca56f5993d16ae0538af9ee8b/src/test/setup.ts#L8), but I could just create a config file for that as well).

I'm thinking it might be a good idea to dump the params and instead allow the config json to be passed in via STDIN.  

The cli params just complicate things and make it harder to add new features.
